.dockerignore

```diff
@@ -1,5 +1,4 @@
 node_modules
-package-lock.json
 *.log
```

Comment:
including package-lock in the docker file will make our builds slightly more predicable